### ISSUE TITLE
Postgres

### ISSUE THREAD

**Issue by ozzyst** [m1k1o/blog#29]
Hello! Thanks for your job the blog is nice! Could you add support of PostgreSQL?

**Comment by m1k1o** [comment]
Should not be very different from existing MySQL implementation. Maybe just some minor syntax changes.

**Comment by ozzyst** [comment]
I hope you will do it :) 

I have a big DB with data from my apps, and don't want to use an extra db. Thanks!

**Comment by error0x001** [comment]
`+`

**Comment by m1k1o** [comment]
@ozzyst @error0x001 available in latest image.

**Comment by ozzyst** [comment]
Thanks a lot! But it doesn't work from the docker-compose example

`SQLSTATE[42P01]: Undefined table: 7 ERROR: relation "posts" does not exist LINE 5: FROM "posts" ^`

Maybe I did something wrong?

**Comment by ozzyst** [comment]
Also, I've checked DB in docker, and there is empty.

UPD. if I create and tables DB by meslf, it will work. I'm not sure that's is ok.

**Comment by m1k1o** [comment]
Yes, tables are not created by the app. It is automatically created by postgres `- ./app/db/postgres:/docker-entrypoint-initdb.d:ro`. So you need to have them downloaded and mounted.

**Comment by amlamarra** [comment]
Hi, I'm seeing this issue as well. I'm getting the error
```plaintext
SQLSTATE[42P01]: Undefined table: 7 ERROR: relation "posts" does not exist LINE 5: FROM "posts" ^
```

What do you mean by "automatically created by postgres"? And what do you mean when you say, "need to have them downloaded and mounted"? Sorry, I'm new to Docker. Could you provide an example on how to get this to work?

**Comment by m1k1o** [comment]
Every SQL file that is located in `/docker-entrypoint-initdb.d` folder inside postgres container will be executed when first time running postgres - as initialization. That means, you need to have:

- `./app/db/postgres` folder on your server, simply by cloning this repository - having only `docker-compose.yaml` is insufficient.
- that folder needs to be mounted as `- ./app/db/postgres:/docker-entrypoint-initdb.d:ro`.

Basically, if you would `git clone` whole repostiroy and then replacing `docker-compose.yaml` with postgres configuration and starting, it should work. If you only copied `docker-compose.yaml` then not.

But I see this as pointlessly complicated and this chould be done automatically by the app. Would be a good PR.
